source/general/wikicontributing.rst

```diff
@@ -4,64 +4,66 @@
 Contributing to the Wiki
 ========================
 
-Setting up the environment
+Setting Up the Environment
 ==========================
 
-To begin editing the wiki we first need to set our environment up. The prerequisites needed is just `python <https://www.python.org/downloads/>`_.
+To begin editing the Wiki, you first need to your environment. `Python`_ is the only dependency required.
```

Comment:
Change to `To begin editing the Wiki, you first need to set up its environment. Python is the only dependency required.`

Reply:
I think your fits better in this scenario

Reply:
Change to `To begin editing the Wiki, you first need to set up its environment. Python is the only dependency required.`